07-07 23:45:48
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/telegram/ext/_extbot.py", line 340, in _do_post
07-07 23:45:48
return await super()._do_post(
07-07 23:45:48
^^^^^^^^^^^^^^^^^^^^^^^
07-07 23:45:48
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/telegram/_bot.py", line 641, in _do_post
07-07 23:45:48
return await request.post(
07-07 23:45:48
^^^^^^^^^^^^^^^^^^^
07-07 23:45:48
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/telegram/request/_baserequest.py", line 200, in post
07-07 23:45:48
result = await self._request_wrapper(
07-07 23:45:48
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
07-07 23:45:48
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/telegram/request/_baserequest.py", line 381, in _request_wrapper
07-07 23:45:48
raise Conflict(message)
07-07 23:45:48
telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
07-07 23:46:00
2025-07-07 21:46:00,110 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7709050722:AAHlO8wju9uWZGp0KLO2-qga8aKRhqhGzG8/getUpdates "HTTP/1.1 200 OK"
07-07 23:46:10
2025-07-07 21:46:10,241 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7709050722:AAHlO8wju9uWZGp0KLO2-qga8aKRhqhGzG8/getUpdates "HTTP/1.1 200 OK"
07-07 23:46:20
2025-07-07 21:46:20,357 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7709050722:AAHlO8wju9uWZGp0KLO2-qga8aKRhqhGzG8/getUpdates "HTTP/1.1 200 OK"
07-07 23:46:23
2025-07-07 21:46:23,457 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7709050722:AAHlO8wju9uWZGp0KLO2-qga8aKRhqhGzG8/getUpdates "HTTP/1.1 409 Conflict"
07-07 23:46:23
2025-07-07 21:46:23,458 - telegram.ext.Updater - ERROR - Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
07-07 23:46:23
2025-07-07 21:46:23,458 - telegram.ext.Updater - ERROR - Exception happened while polling for updates.
07-07 23:46:23
Traceback (most recent call last):
07-07 23:46:23
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/telegram/ext/_updater.py", line 688, in _network_loop_retry
07-07 23:46:23
if not await action_cb():
07-07 23:46:23
^^^^^^^^^^^^^^^^^
07-07 23:46:23
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/telegram/ext/_updater.py", line 384, in polling_action_cb
07-07 23:46:23
raise exc
07-07 23:46:23
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/telegram/ext/_updater.py", line 373, in polling_action_cb
07-07 23:46:23
updates = await self.bot.get_updates(
07-07 23:46:23
